### PR TITLE
Update ECOSYSTEM.md to reflect updated partnerships

### PR DESCRIPTION
**Pull request by stmoy**
## Summary

To help reflect how we've collectively been thinking about partnership and engagement on React Native, this change moves Facebook into the Partners list alongside other partners like Expo, Microsoft, and others. This change helps establish that the React Native teams across Facebook & Microsoft are one big happy family :) 

## Changelog

[General] [Changed] - Updated ECOSYSTEM.md to reflect partnerships between Facebook, Microsoft, and others

## Test Plan

N/A - document change

### CODE REVIEW

**Comment by stmoy** [comment]
CC: @TheSavior

File: ECOSYSTEM.md [hunk 1]
```diff
@@ -10,14 +10,15 @@ There are three types of stakeholders:
...
 
-Our current set of partners include Callstack, Expo, Infinite Red, Microsoft and Software Mansion. Many engineers from these companies are core contributors, and their partner responsibilities also include:
+React Native's current set of partners include Callstack, Expo, Facebook, Infinite Red, Microsoft and Software Mansion. Many engineers from these companies are core contributors, and their partner responsibilities also include:
 
 * **[Callstack](https://callstack.com/):** Manages releases, maintains the [React Native CLI](https://github.com/react-native-community/react-native-cli) and organizes [React Native EU](https://react-native.eu/)
 * **[Expo](https://expo.io/):** Builds [expo](https://github.com/expo/expo) on top of React Native to simplify app development
+* **[Facebook]():** Oversees the React Native product and maintains the [React Native core repo](https://reactnative.dev/)
```
Comment (TheSavior) [inline ECOSYSTEM.md:19]: Hmm, good question about what this should link to. Facebook.com? Facebook's engineering blog? the React Native public docs? Hmm. I don't have a good answer to that. What do you think? 😀

Comment (stmoy) [reply ECOSYSTEM.md:19]: I chose reactnative.dev because it's a public-facing "root" website that points to other endpoints like GitHub/docs/etc.

Comment (rickhanlonii) [reply ECOSYSTEM.md:19]: https://opensource.facebook.com?

**Pull request closed without merging**